cumulusci/tests/pytest_plugins/pytest_sf_vcr.py

```diff
@@ -0,0 +1,75 @@
+import re
+
+from .pytest_sf_orgconnect import sf_pytest_orgname
+
+
+def sf_before_record_cb(request):
+    if request.body and "<sessionId>" in request.body.decode():
+        request.body = re.sub(
+            "<sessionId>.*</sessionId>",
+            "<sessionId>**Elided**</sessionId>",
+            request.body.decode(),
+        ).encode()
+    request.uri = re.sub(
+        "//.*.my.salesforce.com", "//orgname.salesforce.com", request.uri
+    )
+    request.uri = re.sub(
+        "//cs.*.salesforce.com/", "//podname.salesforce.com/", request.uri
```

Comment:
Note that the cs prefix is for sandbox pods, and other prefixes are used for production pods.

Reply:
I'll jut open it up to everything with digits in the domain before the salesforce.com. ".my" URLs will have already been transformed to NOT have digits in the line before.

Reply:
https://github.com/SFDO-Tooling/CumulusCI/pull/1816/files#diff-9b5998118d11aec9e8e2fb607f028f78R25